plugins/modules/ini_file.py

```diff
@@ -46,6 +45,21 @@
       - If left empty, being omitted, or being set to V(null), the O(option) will be placed before the first O(section).
       - Using V(null) is also required if the config format does not support sections.
     type: str
+  section_has:
+    type: list
+    elements: dict
+    required: false
+    suboptions:
+      option:
+        type: str
+        description: the option to look for within the section
+      value:
+        type: str
+        description: locate the section with this specific value
+    description:
+      Among possibly multiple sections of the same name, select the one that contains these values.
+      - with O(state=present), if a suitable section is not found, a new section will be added, including the required options.
+      - with O(state=absent), at most one O(section) is removed if it contains the values
```

Comment:
Linebreaks will be removed in a paragraph, so better make this three paragraphs.
```suggestion
      - Among possibly multiple sections of the same name, select the one that contains these values.
      - With O(state=present), if a suitable section is not found, a new section will be added, including the required options.
      - With O(state=absent), at most one O(section) is removed if it contains the values.
```